app/templates/_babel.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    'presets': [
+        ['@babel/env', {
+            'modules': false,
+            'useBuiltIns': 'usage'
+        }]
+    ],
+    'plugins': [
+        '@babel/plugin-transform-modules-commonjs',
+        "@babel/plugin-proposal-object-rest-spread",
```

Comment:
`@babel/plugin-proposal-object-rest-spread` plugin could be removed, because the proposal [has been finished](https://github.com/tc39/proposals/blob/master/finished-proposals.md) and  `@babel/preset-env` will take care of it, hopefully 😄